dashboard/app/views/plc/enrollment_unit_assignments/show.html.haml

```diff
@@ -26,5 +28,5 @@
     All learning resources
   - @enrollment_unit_assignment.plc_course_unit.get_all_possible_learning_resources.each do |learning_resource|
     %i{class: 'fa ' + (learning_resource.icon.nil? ? 'fa-book' : learning_resource.icon)}
-    = learning_resource.name
+    = link_to learning_resource.name, "http://#{learning_resource.resource_url}", target: '_blank'
```

Comment:
Hmm. I don't want to worry about content creators forgetting the http and us thinking its a relative path by mistake.

I think the smart thing here is to handle this on form submission of the task. If we get a URL that looks like an external link (say begins with www) then prepend the http://. Then, remove the http:// from this line.